yb-voyager/src/tgtdb/postgres.go

```diff
@@ -0,0 +1,807 @@
+/*
```

Comment:
File a ticket to factor out common code between PG and YB.

Reply:
Filed https://yugabyte.atlassian.net/browse/DB-9381